tests/virt_autotest/guest_installation_run.pm

```diff
@@ -33,6 +34,7 @@ sub get_script_run {
     my $parallel_num = get_var("PARALLEL_NUM", "2");
 
     $pre_test_cmd = $pre_test_cmd . " -f " . $guest_pattern . " -n " . $parallel_num . " -r ";
+    $pre_test_cmd .= " 2>&1 | tee /tmp/s390x_guest_install_test.log" if (is_s390x);
```

Comment:
Why do you need this ?  Is this temporary or permanent change ?

Reply:
Refer to current test logic, svrit backend -> lpar_cmd-> /usr/share/qa/tools/test_virtualization-virt_install_withopt-run -> /usr/share/qa/virtautolib/lib/virt-install.sh ... Generally, we totally do not clear what happen during guest installation test run on s390x LPAR. So, create /tmp/s390x_guest_install_test.log that is useful to us to debug new test failure for s390x guest installation test failure